### ISSUE TITLE
Add control for selecting shape encoding

### ISSUE THREAD

**Issue by metasoarous** [matsengrp/olmsted#35]
This will be much the same work as for #32 and especially #34, as we'd likely prioritize for nominal and ordinal variables, though we could also possibly add a processing step that looks for natural range boundaries for quantitative values.

**Comment by eharkins** [comment]
Not prioritized because we already have a separate issue for the same thing but controlling color by field.

**Comment by eharkins** [comment]
The reason this was done before v1 priority issues is because it's just a matter of copy and paste and changing 3 lines after #32 was done
